ccl/streamingccl/streamingest: TestStreamingZoneConfigsMismatchedRegions failed

ccl/streamingccl/streamingest.TestStreamingZoneConfigsMismatchedRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/208e17f4-706c-4579-a097-bd3cd7e0c0d0?testReportRun=2&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL3N0cmVhbWluZ2NjbC9zdHJlYW1pbmdlc3Q6c3RyZWFtaW5nZXN0X3Rlc3Q=) on master @ [5f76d93b2ad8d29c14c9b176753ec33cbf81e3fe](https://github.com/cockroachdb/cockroach/commits/5f76d93b2ad8d29c14c9b176753ec33cbf81e3fe):


```
=== RUN   TestStreamingZoneConfigsMismatchedRegions
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestStreamingZoneConfigsMismatchedRegions3783793182
    test_log_scope.go:81: use -show-logs to present logs inline
    testutils.go:468: condition failed to evaluate within 45s: from testutils.go:497: stream ingestion has not recorded any progress yet, waiting to advance pos 1702976119.762838544,0
    panic.go:523: -- test log scope end --
test logs left over in: outputs.zip/logTestStreamingZoneConfigsMismatchedRegions3783793182
--- FAIL: TestStreamingZoneConfigsMismatchedRegions (271.33s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamingZoneConfigsMismatchedRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34744

--- a/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
+++ b/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
@@ -1275,7 +1275,7 @@ func TestStreamingRegionalConstraint(t *testing.T) {
 func TestStreamingMismatchedMRDatabase(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.UnderRace(t, "takes too long under stress race")
+	skip.UnderDuress(t, "multi node c2c is very flaky")
 
 	ctx := context.Background()
 	regions := []string{"mars", "venus", "mercury"}
@@ -1356,7 +1356,7 @@ WHERE
 func TestStreamingZoneConfigsMismatchedRegions(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.UnderRace(t, "takes too long under stress race")
+	skip.UnderDuress(t, "multi node c2c is very flaky")
 
 	ctx := context.Background()
 	regions := []string{"mars", "venus", "mercury"}

